src/dao/mongo/base.ts

```diff
@@ -20,8 +20,13 @@ export default class BaseMongoDAO<
     super(loggerFactory);
   }
 
+  public async findById(id: string): Promise<Model> {
+    return await this.collection().findOne({ id });
+  }
+
   public async create(createOptions: CreateOptions): Promise<Model> {
     const model: Model = {
+      // TODO: mongo automatically generates an id that is indexed. Maybe we should use that instead?
```

Comment:
Not a bad idea. My only concern would be about relying on a platform specific identifier instead of one the we control, for the crazy future where we migrate away from mongo and keep the dao interface the same.